packages/web-components/tests/e2e-storybook/cypress/integration/masthead/masthead-l1.e2e.js

```diff
@@ -183,4 +184,38 @@ describe('cds-masthead | with L1 (desktop)', () => {
       })
   });
 
+  it('should have horizontal scroll for L1 items working', () => {
+    cy.viewport(1100, 780);
+    
+    cy.get(_selectors.l1)
+      .shadow()
+      .find(_selectors.l1ScrollNextArrow)
+      .click()
+      .get(_selectors.l1)
+      .shadow()
+      .find(_selectors.l1Item)
+      .eq(5)
+      .should('be.visible');
+  });
+  
+});
+
+describe('dds-masthead L1 | default (mobile)', () => {
```

Comment:
```suggestion
describe('c4d-masthead L1 | default (mobile)', () => {
```

Reply:
Theres a big mix of prefixes used for these titles. Make them all consistent while we're here?